MeetupPlatformApi/Features/Meetups/DeleteMeetup/DeleteMeetupFeature.cs

```diff
@@ -24,6 +24,7 @@ public class DeleteMeetupFeature : FeatureBase
     [Authorize(Roles = Roles.Organizer)]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
```

Comment:
If you add a **ProducesResponseType**, then you need to specify a "response code" comment for this type. I advise you to ask Bahdan about the need to specify this **ProducesResponseType** in this controller

Reply:
We don't need to specify `401 Unauthorized` and `403 Forbidden` response types if we use the `[Authorize]` attribute, since these responses are added automatically to the Swagger specification.